frontend/public/components/modals/delete-pvc-modal.tsx

```diff
@@ -72,6 +79,7 @@ const DeletePVCModal = withHandlePromise<DeletePVCModalProps>((props) => {
 
 export type DeletePVCModalProps = {
   pvc: PersistentVolumeClaimKind;
+  kind: K8sKind;
```

Comment:
This should always be `PersistentVolumeClaimModel` right? Do we need to pass `kind` as a prop?

Reply:
> This should always be `PersistentVolumeClaimModel` right? Do we need to pass `kind` as a prop?

Yes, it will. Oh I see! No. 
Will use 'PersistentVolumeClaimModel` instead of passing `kind` for `resourceListPathFromModel` method. 
Thanks for the catch :)

Reply:
> This should always be `PersistentVolumeClaimModel` right? Do we need to pass `kind` as a prop?

Updated.